velocity/src/main/java/me/leoko/advancedban/velocity/listener/ChatListenerVelocity.java

```diff
@@ -0,0 +1,21 @@
+package me.leoko.advancedban.velocity.listener;
+
+import com.velocitypowered.api.event.PostOrder;
+import com.velocitypowered.api.event.Subscribe;
+import com.velocitypowered.api.event.player.PlayerChatEvent;
+import me.leoko.advancedban.Universal;
+
+public class ChatListenerVelocity {
+
+    @Subscribe(order = PostOrder.FIRST)
+    public void onChat(PlayerChatEvent event) {
+        if (!event.getMessage().startsWith("/")) {
```

Comment:
Velocity is not bungeecord, there is a separate event for command execution, you will never get a command execution using the PlayerChatEvent